cdp_scrapers/legistar_utils.py

```diff
@@ -395,6 +395,43 @@ class Status(enum.IntEnum):
     uris: Optional[List[ContentURIs]] = None
 
 
+def parse_video_page_url(video_page_url: str, client: str) -> List[ContentURIs]:
```

Comment:
The return type is actually `Optional[List[ContentURIs]]`, which you have correctly listed in the docstring. This is so minor, so I'm going to merge this in.